pkg/cache/cache.go

```diff
@@ -73,36 +75,61 @@ func init() {
 
 func (c *serviceAccountCache) Get(name, namespace string) (role, aud string, useRegionalSTS bool, tokenExpiration int64) {
```

Comment:
Can we add a comment for `Get()` and also something to the README that explains the ConfigMap and precedence between CM/SA?